assets/style/about/about.scss

```
@@ -6,9 +6,536 @@
+        @media (max-width: $tablet-l) {
+          margin-bottom: 40px;
+        }
+        @media (max-width: $tablet-l) {
+          font-size: 38px;
+        }
+        @media (max-width: $phone-m) {
+          font-size: 32px;
+        }
+      }
+      .main__wr-principles-box {
+        display: grid;
+        grid-template-columns: 510px 510px 510px;
+        grid-template-rows: 275px 275px;
+        gap: 35px;
+        margin-bottom: 146px;
+        @media (max-width: 1650px) {
+          grid-template-columns: minmax(350px, 1fr) minmax(350px, 1fr) minmax(
+              350px,
+              1fr
+            );
+        }
+        @media (max-width: $pc-s) {
+          grid-template-columns: minmax(250px, 1fr) minmax(250px, 1fr);
+          grid-template-rows: minmax(290px, 1fr) minmax(290px, 1fr) minmax(
+              290px,
+              1fr
+            );
+          margin-bottom: 0px;
+        }
+        @media (max-width: $tablet-l) {
+          grid-template-columns: minmax(250px, 1fr);
+          grid-template-rows:
+            minmax(210px, 1fr) minmax(210px, 1fr) minmax(210px, 1fr)
+            minmax(210px, 1fr) minmax(210px, 1fr) minmax(210px, 1fr);
+          gap: 25px;
+        }
+
+        .main__principles-box {
+          background-color: #f9fafd;
+          padding-left: 41px;
+          font-family: $font;
+          @media (max-width: $tablet-l) {
+            max-width: 100%;
+            text-align: center;
+            justify-items: center;
+            padding-left: 0px;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+          }
+
+          img {
+            width: 50px;
+            padding-top: 35px;
+            margin-bottom: 26px;
+            @media (max-width: $phone-l) {
+              padding-top: 10px;
+              margin-bottom: 10px;
+            }
+          }
+          .main__principles-box-title {
+            margin-bottom: 7px;
+            font-weight: 700;
+            font-size: 22px;
+            line-height: 32px;
+            color: $text-black-color;
+            @media (max-width: $pc-s) {
+              max-width: 70%;
+              font-size: 20px;
+            }
+            @media (max-width: $phone-m) {
+              font-size: 16px;
+            }
+          }
+          .main__principles-box-text {
+            display: block;
+            max-width: 390px;
+            font-weight: 400;
+            font-size: 20px;
+            line-height: 26px;
+            color: $text-blue-color;
+            @media (max-width: $pc-s) {
+              max-width: 70%;
+              font-size: 18px;
+            }
+            @media (max-width: $phone-m) {
+              font-size: 14px;
+              max-width: 90%;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  &__wr-certificates {
+    background-image: url(../../images/about/blue-crystal-background.png);
+    min-height: 1313px;
```

Это просто самоубийственно. Представь, что будет, если по какой-то причине часть контента внутри этого блока не прогрузится или динамически уменьшится после выполнения какого-нибудь скрипта. Мы будем иметь дело с тысячей пикселей пустого места на странице. Я крайне не рекомендую так делать. Контент сам растянется на нужную высоту. А снизу можно будет поставить отступ (например, margin-bottom, ну или padding-bottom, если нам важно сохранить кусочек фоновой картинки снизу).
```suggestion
padding-bottom: 60px;
```